source/manual/on-call.html.md

```diff
@@ -23,6 +23,22 @@ to keep GOV.UK running at night, on the weekends and on public holidays.
 - Logs are not as important as being available - if you need to lose some logs
   in order to bring the site back up, that's probably a good trade-off to make
 
+## On call checklist
+
+You should do these things before going on call so you're prepared.
+
+1. Have the numbers of other people on your shift saved in your phone. This
+includes whoever is on Escalations. Get these numbers from PagerDuty.
+2. Ensure you have an up to date local copy of the [Developer Docs][docs] repository
+and that you can build it.
+3. Make sure your [`fabric-scripts`][fabric] are up to date.
+4. Make sure you can VPN to the office or disaster recovery location
+5. Read these documents:
+    - [Stopping Nginx on the cache machines to fail to our static mirror](https://github.com/alphagov/fabric-scripts/blob/master/incident.py)
```

Comment:
Is this still a thing people should know about?

Reply:
Changed for the generic "fallback to static mirror" advice page